src/engraving/dom/select.cpp

```diff
@@ -711,14 +713,16 @@ void Selection::updateSelectedElements()
         if (sp->isVolta()) {
             continue;
         }
-        if (sp->isSlur()) {
+        if (sp->isSlur() || sp->isHairpin()) {
             // ignore if start & end elements not calculated yet
             if (!sp->startElement() || !sp->endElement()) {
                 continue;
             }
             if ((sp->tick() >= stick && sp->tick() < etick) || (sp->tick2() >= stick && sp->tick2() < etick)) {
```

Comment:
why `sp->tick2() < etick` here, but `sp->tick2() <= etick` further down? Smells like an off-by-one error to me